solutions/izho-17-LongestBeautifulSequence.mdx

```diff
@@ -88,6 +87,23 @@ $$
 \end{align}
 $$
 
+And we want some number AND $\texttt{cur} = 8$.

```

Comment:
8? why 8?

Reply:
oh it's not late to find , it should be 9